src/DMG.cpp

```diff
@@ -68,12 +69,12 @@ bool DMG::tick() {
 	return true;
 }
 
-bool DMG::tick_frame() {
+bool DMG::tickFrame() {
 	do {
 		tick();
-	} while (!ppu.frame_complete);
+	} while (!ppu.frameComplete);
```

Comment:
I want to revisit the frameComplete logic and make things more cycle accurate. At the very least I think I might want to add a complete() function to mirror CPU operation and access, I'd like to keep all PPU state stuff internal to the PPU.